lib/kwtool_web/endpoint.ex

```diff
@@ -29,7 +29,7 @@ defmodule KwtoolWeb.Endpoint do
     at: "/",
     from: :kwtool,
     gzip: false,
-    only: ~w(css fonts images js favicon.ico robots.txt)
+    only: ~w(css fonts images js sample favicon.ico robots.txt)
```

Comment:
SHouldn't this be generalized to allow `csv` instead of just sample?

Reply:
Hi @olivierobert 
I am not sure what you meant, the context here is allowing Phoenix server the `sample` directory for static sample files, e.g: `priv/static/sample/keywords.csv`.

If we declare a `sample` in this config, then the user can download the file within that folder.

I also research for config allow download file with extension `*csv` but not found any.